lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+#: messages.c:91
+msgid "Fan test"
+msgstr "Fl\xe1kttest"
+
+# MSG_FANS_CHECK c=13
+#: messages.c:33
+msgid "Fans check"
+msgstr "Fl\xe1ktcheck"
+
+# MSG_FSENSOR c=12
+#: messages.c:49
+msgid "Fil. sensor"
+msgstr "Fil. sensor"
+
+# MSG_FIL_RUNOUTS c=15
+#: messages.c:34
+msgid "Fil. runouts"
+msgstr "Fil. avbrott"
+
+# MSG_FILAMENT_CLEAN c=20 r=2
+#: messages.c:37
+msgid "Filament extruding & with correct color?"
+msgstr "Extruderas filament med r\xe1tt f\xe1rg?"
+
+# MSG_NOT_LOADED c=19
+#: ultralcd.cpp:2564
+msgid "Filament not loaded"
+msgstr "Filament ej laddat"
+
+# MSG_FILAMENT_SENSOR c=20
+#: messages.c:97
+msgid "Filament sensor"
+msgstr "Filament sensor"
+
+# MSG_FILAMENT_USED c=19
+#: ultralcd.cpp:2713
+msgid "Filament used"
+msgstr "Anv\xe1nt filament"
+
+# MSG_PRINT_TIME c=19
+#: ultralcd.cpp:2714
+msgid "Print time"
+msgstr "Utskriftstid"
+
+# MSG_FS_ACTION c=10
+#: messages.c:151
+msgid "FS Action"
+msgstr "FS aktion"
+
+# MSG_FILE_INCOMPLETE c=20 r=3
+#: ultralcd.cpp:8634
+msgid "File incomplete. Continue anyway?"
+msgstr "Filen \xe1r ofullst\xe1ndig. Forts\xe1tta \xe1nda?"
+
+# MSG_FINISHING_MOVEMENTS c=20
+#: messages.c:45
+msgid "Finishing movements"
+msgstr "Avslutar flyttning"
+
+# MSG_V2_CALIBRATION c=18
+#: messages.c:124
+msgid "First layer cal."
+msgstr "F\xefrstalager kalib."
+
+# MSG_WIZARD_SELFTEST c=20 r=8
+#: ultralcd.cpp:4942
+msgid "First, I will run the selftest to check most common assembly problems."
+msgstr "F\xefrst kommer jag att utf\xefra sj\xe1lvtestet f\xe1r att kontrollera de vanligaste monteringsproblemen."
+
+# MSG_MMU_FIX_ISSUE c=20 r=4
+#: mmu.cpp:727
+msgid "Fix the issue and then press button on MMU unit."
+msgstr "Atg\xe1rda problemet och tryck sedan pa knappen pa MMU-enheten."
+
+# MSG_FLOW c=15
+#: ultralcd.cpp:6888
+msgid "Flow"
+msgstr "Fl\xefde"
+
+# MSG_SELFTEST_COOLING_FAN c=20
+#: messages.c:88
+msgid "Front print fan?"
+msgstr "Frontfl\xe1kt?"
+
+# MSG_BED_CORRECTION_FRONT c=14
+#: ultralcd.cpp:3116
+msgid "Front side[um]"
+msgstr "Front[um]"
+
+# MSG_SELFTEST_FANS c=20
+#: ultralcd.cpp:8179
+msgid "Front/left fans"
+msgstr "Front/v\xe1nster fl\xe1kt"
+
+# MSG_SELFTEST_HEATERTHERMISTOR c=20
+#: ultralcd.cpp:8127
+msgid "Heater/Thermistor"
+msgstr "V\xe1rmare/Termistor"
+
+# MSG_BED_HEATING_SAFETY_DISABLED c=20 r=4
+#: Marlin_main.cpp:9874
+msgid "Heating disabled by safety timer."
+msgstr "Uppv\xe1rmning avaktiverad av s\xe1kerhetstimer."
+
+# MSG_HEATING_COMPLETE c=20
+#: messages.c:51
+msgid "Heating done."
+msgstr "Uppv\xe1rmning klar."
+
+# MSG_HEATING c=20
+#: messages.c:50
+msgid "Heating"
+msgstr "Uppv\xe1rmning"
+
+# MSG_WIZARD_WELCOME c=20 r=7
+#: messages.c:121
+msgid "Hi, I am your Original Prusa i3 printer. Would you like me to guide you through the setup process?"
+msgstr "Hej, jag \xe1r din Prusa i3-skrivare. Vill du att jag ska guida dig genom installationsprocessen?"
+
+# MSG_FILAMENTCHANGE c=18
+#: messages.c:43
+msgid "Change filament"
+msgstr "\xe1ndra filament"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.